test/message/test_runner_output_spec_reporter.out

```diff
@@ -75,6 +75,33 @@
     operator: 'strictEqual'
   }
 
+ async assertion object fail (*ms)
+  AssertionError [ERR_ASSERTION]: Expected values to be loosely deep-equal:
+  
+  {
+    bar: 1,
+    foo: 1
+  }
+  
+  should loosely deep-equal
+  
+  {
+    bar: 2
+  }
+      *
+      *
+      *
+      *
+      *
+      *
+      * {
+    generatedMessage: true,
+    code: 'ERR_ASSERTION',
+    actual: [Object],
```

Comment:
this is the result of `util.inspect(AssertionError)` when comparing objects and is not related to the test runner